 <robot name="sample_box">
   <link name="sample_box">
     <inertial>
       <origin xyz="-0 0 0" />
       <mass value="1.0" />
       <inertia  ixx="1.0" ixy="0.0"  ixz="0.0"  iyy="100.0"  iyz="0.0"  izz="1.0" />
     </inertial>
     <visual>
       <origin xyz="0 -180 0"/>
       <geometry>
         <box size=".05 .05 .05" />
       </geometry>
     </visual>
     <collision>
       <origin xyz="0 0 0"/>
       <geometry>
         <box size=".05 .05 .05" />
       </geometry>
     </collision>
   </link>
   <gazebo reference="sample_box">
     <material>Gazebo/Blue</material>
   </gazebo>
 </robot>
